packages/qjackcapture/PKGBUILD

```diff
@@ -0,0 +1,26 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_project=QJackCapture
+pkgname="${_project,,}"
+pkgver=0.1.2
+pkgrel=1
+pkgdesc='A GUI for easy recording of JACK audio sources using jack_capture'
+arch=(any)
+url='https://github.com/SpotlightKid/qjackcapture'
+license=(GPL)
+depends=(hicolor-icon-theme jack_capture python-pyjacklib python-pyqt5)
```

Comment:
```suggestion
depends=(hicolor-icon-theme jack_capture python-natsort python-pyjacklib python-pyqt5)
```
was missing so I got this error:
```
Traceback (most recent call last):
  File "/usr/bin/qjackcapture", line 33, in <module>
    sys.exit(load_entry_point('QJackCapture==0.1.2', 'console_scripts', 'qjackcapture')())
  File "/usr/bin/qjackcapture", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 171, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/lib/python3.10/site-packages/qjackcapture/app.py", line 47, in <module>
    from natsort import humansorted
ModuleNotFoundError: No module named 'natsort'
```

Reply:
In this case, the dependency was there since the initial release, but apparently my AUR PKGBUILD was wrong. :(

Some kind of smoke test in `check()` might make sense, but just running the program with `-h` seems weird. It wouldn't catch all missing imports in any Python program, since some imports might be deferred or not done when only running the command line. In the case of `qjackcapture` it would work, though, yes. I'll have to consider this a bit.